motion/motionprocessor.go

```diff
@@ -86,12 +87,20 @@ type RecordingListener interface {
 	RecordingEnded()
 }
 
+func (mp *MotionProcessor) LastFrame() *cptvframe.Frame {
+	return mp.lastFrame
+}
+func (mp *MotionProcessor) setLastFrame(frame *cptvframe.Frame) {
+	mp.lastFrame = frame
+}
+
```

Comment:
I believe that the frame loop already has a way of retrieving the most recent frame